models/migrate_test.go

```diff
@@ -32,8 +32,9 @@ func TestMigrate_InsertMilestones(t *testing.T) {
 	unittest.CheckConsistencyFor(t, &Milestone{})
 }
 
-func assertCreateIssues(t *testing.T, reponame string, isPull bool) {
+func assertCreateIssues(t *testing.T, isPull bool) {
 	assert.NoError(t, unittest.PrepareTestDatabase())
+	reponame := "repo1"
```

Comment:
While this is now redundant, we might need to add tests later for other repo's.

Reply:
Agreed. There is [another pull request](https://github.com/go-gitea/gitea/pull/18621) adding more testing for repo dump/restore and preparing the ground for adding verification for other repositories.